x-pack/plugins/observability_solution/uptime/common/rules/uptime_rule_field_map.ts

```diff
@@ -5,7 +5,9 @@
  * 2.0.
  */
 
-export const uptimeRuleFieldMap = {
+import { FieldMap } from '@kbn/alerts-as-data-utils';
```

Comment:
This looks exactly the same as the `synthetics_rule_field_map`. Maybe this can be moved to a common location and reused? Is it expected that the two field maps will diverge in the future?

Reply:
ideally they should be. i guess could be just moved to synthetics plugin, but it's kinda tricky as well. Since one of the plugin being disabled would cause issues.